multicluster/cmd/install.go

```diff
@@ -231,12 +227,24 @@ func buildMulticlusterInstallValues(ctx context.Context, opts *multiclusterInsta
 	defaults.Gateway.Port = opts.gateway.Port
 	defaults.Gateway.Probe.Seconds = opts.gateway.Probe.Seconds
 	defaults.Gateway.Probe.Port = opts.gateway.Probe.Port
-	defaults.IdentityTrustDomain = values.IdentityTrustDomain
 	defaults.LinkerdNamespace = controlPlaneNamespace
-	defaults.ProxyOutboundPort = uint32(values.Proxy.Ports.Outbound)
 	defaults.LinkerdVersion = version.Version
 	defaults.RemoteMirrorServiceAccount = opts.remoteMirrorCredentials
 	defaults.Gateway.ServiceType = opts.gateway.ServiceType
 
+	if ignoreCluster {
```

Comment:
`ignoreCluster` will need to be passed in as an argument, but that's fine if we can avoid using it as a global.

Reply:
Done in https://github.com/linkerd/linkerd2/pull/7564/commits/747061fdca1ebf0626a2edb93babfdb804c51f67